src/views/integrations/event-tickets/emails/template-parts/header/head/tec-styles.php

```diff
@@ -174,4 +174,11 @@
 	h3.tec-tickets__email-table-content-event-title a {
 		text-decoration: none;
 	}
+	
+	.tec-tickets__email-table-content-order-event-title,
```

Comment:
I think we should not use different selectors for the `Same Event Title` in different emails. As, this will keep making the styles tag long and these would be carried over all emails even if not used. 

cc: @codingmusician @juanfra

Reply:
Agree.

Reply:
So what do we do in the case where the styling for the event title is different? The design for the event title in the Completed Order and Purchase Receipt emails is different than the styling for the Event Title in the Ticket and RSVP emails. This is why I was differentiating the class as the "order" event title.

Reply:
Sorry, I misread the initial comment. We _should_ use different class names. My bad, sorry for that!